agent/config/config.go

```diff
@@ -177,6 +177,9 @@ const (
 	ContainerInstancePropagateTagsFromEC2InstanceType
 )
 
+// TODO: update path once finalized. using guardDuty.service as a temporary placeholder
+const GuardDutyServiceFilePath = "/usr/lib/systemd/system/guardDuty.service"
```

Comment:
Q. should this path defined in [config_unix.go](https://github.com/aws/amazon-ecs-agent/blob/master/agent/config/config_unix.go)?

Reply:
moved it to [config_unix.go](https://github.com/aws/amazon-ecs-agent/blob/master/agent/config/config_unix.go)